can sniff check buillds and runs an C binary, which breaks configure in
cross-compile environ.
This patch simple disables it.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Inappropriate [embedded specific]

Index: gdk-pixbuf-2.22.1/configure.ac
===================================================================
--- gdk-pixbuf-2.22.1.orig/configure.ac	2010-11-25 16:26:41.000000000 +0800
+++ gdk-pixbuf-2.22.1/configure.ac	2010-11-25 16:29:22.000000000 +0800
@@ -99,6 +99,8 @@
 AC_MSG_CHECKING([for native Win32])
 LIB_EXE_MACHINE_FLAG=X86
 EXE_MANIFEST_ARCHITECTURE=X86
+dnl disable can sniff check in cross compile
+gio_can_sniff=no
 case "$host" in
   *-*-mingw*)
     os_win32=yes
